sunpy/map/tests/test_mapbase.py

```diff
@@ -941,20 +940,6 @@ def test_rotate(aia171_test_map):
     assert aia171_test_map_crop_rot.data.shape[0] < aia171_test_map_crop_rot.data.shape[1]
 
 
-@pytest.mark.xfail(version.parse(np.__version__) >= version.parse("1.2.0"),
```

Comment:
I decided to remove this test.

We can't get this warning anymore due to a numpy change, so either we make the function error in this case or we let numpy error and users deal with it?